src/styles/components/multiple-form.less

```diff
@@ -137,12 +136,14 @@ base/forms.less
         flex: 1 1;
         padding-bottom: @base-spacing-unit;
         padding-right: 0;
+        padding-left: 0;
         padding-top: @base-spacing-unit;
```

Comment:
Since we're defining all four `padding` values, let's use the shorthand: `padding: @base-spacing-unit 0;`